### DISCUSSION TITLE
Incorrect time in docker installation

### DISCUSSION THREAD

**Discussion by mauroreggio** [pglombardo/PasswordPusher#2342] [Q&A] [answered]
Hi all.
Someone noticed, like me, that the time in the docker container of passwordpusher is not set correctly?
I use `PWP__TIMEZONE='Europe/Rome'` in my .env file, but into the container i'm at UTC (2 hours back)

`ac2d3:/opt/PasswordPusher$ date`
`Thu Jul 11 18:20:43 UTC 2024`

`ac2d3:/opt/PasswordPusher$ date +"%Z %z"`
`UTC +0000`

Not know if matter, but usually in Alpine Linux (i think the container is based on Alpine) there must be a `/etc/localtime` file linked to the corresponding TimeZone file in `/usr/share/zoneinfo/` for correct TimeZone setting, ... but into the container there is not `/etc/localtime` file. Anyway  `/usr/share/zoneinfo/Europe/Rome` file exist into the container. Maybe I'm wrong and I'm off track.
I desire fix only because this make my Dashboard and Admin log have a wrong time.
Any help is apreciated.
Thanks.

**Comment by pglombardo** [comment, marked as answer]
Hi @mauroreggio - Just a few hours ago I release [v1.41.15](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.41.15) which fixes the timezone in the Admin dashboard.  Could you try that version out and see if it resolves the issue?

**Reply by mauroreggio** [reply]
Hi @pglombardo i do now what you suggest.
Seems that the container time stay in UTC

`a81a707:/opt/PasswordPusher$ bin/pwpush console`
`Password Pusher Version: 1.41.15`
`Loading production environment (Rails 7.1.3.4)`
`irb(main):001> exit`
`a81a707:/opt/PasswordPusher$ date`
`Mon Jul 15 06:38:44 UTC 2024`

But now the logtime into PasswordPusher dashboard is recorded in my timezone time (Europe/Rome)

![immagine](https://github.com/user-attachments/assets/d30ca6fb-d995-468f-8706-d88095147119)

So, for me is solved, seems that in this new release PasswordPusher use the correct timezone.
Thanks again.
Mauro.

**Reply by pglombardo** [reply]
Excellent.  I'm glad that fixed it for you.  If you have any other issues let me know.
